### ISSUE TITLE
[BUG] iOS app always loads non-compressed videos

### ISSUE THREAD

**Issue by raisinbear** [immich-app/immich#1025]
**Describe the bug**
Immich reencodes videos into small files for previews. The web interface uses / plays those reencoded files. The iOS app, however, always loads the - usually much larger - original file. I do not know, but assume this is not the intended behavior. For instance, the creators left a note in the iOS app settings panel that an optional three-stage loading process significantly increases data usage. So I conclude that data usage optimization is indeed a factor that attention is being paid to. I only noticed this after wondering why the Immich app had used quite a lot of data recently.

**Task List**

*Please complete the task list below. We need this information to help us reproduce the bug or point out problems in your setup. You are not providing enough info may delay our effort to help you.*

- [ x ] I have read thoroughly the README setup and installation instructions.
- [ x ] I have included my `docker-compose` file.
- [ x ] I have included my redacted `.env` file.
- [ x ] I have included information on my machine, and environment.

[docker-compose.yml.txt](https://github.com/immich-app/immich/files/10098521/docker-compose.yml.txt)
[env.txt](https://github.com/immich-app/immich/files/10098523/env.txt)

**To Reproduce**
1. Make sure a video exists in the library.
2. Start a file handle monitoring software like e.g. `fatrace` on the server.
3. Compare which files are being read while playing back the video through either web interface or iOS app.
4. Note that when playing back through iOS app, only the non-reencoded original file is accessed.

**Expected behavior**
iOS app accesses reencoded video file to reduce data usage and only requests the original, if the user downloads the video to their library.

**Screenshots**
n/a

**System**
- Phone OS [iOS, Android]: `iOS 16.1.1`
- Server Version: `1.36.2`
- Server OS: raspbian buster (32-bit)
- Mobile App Version: `1.36.1 build.72`

**Additional context**

**Comment by alextran1502** [comment]
It was the original behavior of the app, and we haven't circled back to implement playing the encoded file. Thank you for reporting

**Comment by alextran1502** [comment]
Digging into this issue and found out that mp4 transcoded from the MOV file can play on iOS but doesn't have sound 🤔. The same file is played on the web and has sound. Not sure where the issue is.

**Comment by raisinbear** [comment]
Just did some research and this led me on the right track: https://stackoverflow.com/questions/57383307/no-audio-when-playing-html5-video-on-ios/57396013#57396013
At least I believe so, since I don't know what playback engine is used in immich and cannot debug the iOS version.
For my test I did the next best thing and linked the encoded videos folder into a filebrowser instance (a web file browser atop an arbitrary base folder on the server) and tried to playback the video. As you said - no sound. I then changed the encoding preference for audio from mp3 to aac and at least it works in that scenario.
